frontend/app/src/components/display/BalanceDisplay.vue

```diff
@@ -71,12 +62,16 @@ export default defineComponent({
 });
 </script>
 
-<style module lang="scss">
-.gain {
-  color: #4caf50 !important;
-}
+<style scoped lang="scss">
+.balance-display {
+  line-height: 1.5em;
 
-.loss {
-  color: #d32f2f !important;
+  &--gain {
+    color: #4caf50 !important;
+  }
+
+  &--loss {
+    color: #d32f2f !important;
+  }
```

Comment:
Can we revert back to the CSS module style? In general for new components and when refactoring if it is possible we should prefer CSS modules.